integration/mock-backend-server/src/main/resources/Dockerfile

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 # -----------------------------------------------------------------------
 
-FROM adoptopenjdk/openjdk11:jre-11.0.9_11.1-alpine
+FROM adoptopenjdk/openjdk11:jre-11.0.14.1_1
```

Comment:
This image supports M1 macs. Please check whether this change is acceptable to continue.
https://hub.docker.com/r/adoptopenjdk/openjdk11/tags?page=1&name=jre-11.0.14.1_1

Reply:
+1 from me since this is the mock backend, even though this is an Ubuntu image